test/installation_node/install with --alias

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+
+die () { echo "$@" ; exit 1; }
+
+\. ../../nvm.sh
+
+NVM_DEBUG=1 nvm install --alias=9 9.11.2 \
+  | grep '9 -> 9.11.2 (-> v9.11.2)' || die 'did not call through to nvm_make_alias'
```

Comment:
this test is failing; i think maybe because it's grepping the install output instead of letting the install complete, and running `nvm alias | grep`?